llvm/include/llvm/IR/PassManager.h

```diff
@@ -519,7 +519,7 @@ class PassManager : public PassInfoMixin<
 
       PreservedAnalyses PassPA = Pass->run(IR, AM, ExtraArgs...);
       if (printPassNames()) {
-        llvm::outs() << "*** Pass: " << Pass->name() << " ***\n";
```

Comment:
I had to change this to make the aatrace and this print to the same output, s.t. I can assign the proper queries to the passes.